### PR TITLE
fix(data): align metadata with dataset paths

### PR DESCRIPTION
**Pull request by Marigold**
Manual fix for https://github.com/owid/etl/issues/793. It's likely we'll solve this in a different more automatic way.

* covid dataset changed short name from `covid19` to `covid` and version from today's date to `latest` to match dataset path (we still have `data_accessed` from source if anyone wants to check the date of latest update)
* grapher datasets that changed `short_name` to match their path will have to be updated in production DB

### CODE REVIEW

File: etl/steps/data/grapher/cait/2022-08-10/carbon_dioxide_emissions_by_sector.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from owid import catalog
+
```
Comment (pabloarosado) [inline etl/steps/data/grapher/cait/2022-08-10/carbon_dioxide_emissions_by_sector.py:2]: Isn't this dataset identical to `cait/2022-08-10/carbon_dioxide_emissions_by_sector`? I guess your plan was to remove `co2_emissions`, but you forgot?

Comment (pabloarosado) [reply etl/steps/data/grapher/cait/2022-08-10/carbon_dioxide_emissions_by_sector.py:2]: Or, alternatively, for consistency with `ch4_emissions` and `n2o_emissions`, why not removing `carbon_dioxide_emissions_by_sector`? Maybe I'm misunderstanding the problem.

**Review by pabloarosado — APPROVED** [review]
Thanks for fixing all this. It looks good (except that, as I commented below, I think file `carbon_dioxide_emissions_by_sector` should not exist, but I'm not sure if I'm missing something). But I'm not sure if there could be unintended changes with this big refactor. Can you think of any?

File: etl/steps/data/grapher/cait/2022-08-10/carbon_dioxide_emissions_by_sector.py (hunk 1 above)
Comment (Marigold) [reply etl/steps/data/grapher/cait/2022-08-10/carbon_dioxide_emissions_by_sector.py:2]: My bad, I originally thought I'd rename those files to keep `short_name` same as in our DB, but then I changed my mind and forgot to delete it.

**Comment by Marigold** [comment]
[Datadiff](https://buildkite.com/our-world-in-data/etl-build-public-datasets-branch/builds/1214#0185e349-e21f-462b-b724-8a6d55a22e15) looks ok. It's gonna change metadata in our data catalog, but I don't think anyone would notice. We definitely have to change grapher short names in both production and staging. I can't think of anything other than that.

**Pull request merged**